### PR TITLE
Added FastMC.net

### PR DESCRIPTION
**Pull request by thechriscraft**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by thechriscraft** [comment]
Updated manifest.json...

**Comment by thechriscraft** [comment]
> **Servername has to be directory name!**
> 
> Please fix the issues by pushing **one** commit to the pull request to prevent too many automatic reviews.

manifest.json changed...

File: minecraft_servers/fastmc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "server_name": "fastmc",
+  "nice_name": "FastMC Network",
+  "direct_ip": "play.FastMC.net",
```
Comment (JakobBruening) [inline minecraft_servers/fastmc/manifest.json:4]: Please use lowercase.

File: minecraft_servers/fastmc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "server_wildcards": [
+    "%.fastmc.net"
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://www.FastMC.net/",
```
Comment (JakobBruening) [inline minecraft_servers/fastmc/manifest.json:12]: Please use lowercase (and remove the slash at the end).

File: minecraft_servers/fastmc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "web_shop": "https://shop.FastMC.net/",
```
Comment (JakobBruening) [inline minecraft_servers/fastmc/manifest.json:13]: Please use lowercase (and remove the slash at the end).

File: minecraft_servers/fastmc/manifest.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "twitter": "FastMC_NET",
+    "tiktok": "@FastMCNetwork",
+    "youtube": "https://FastMC.net/sonstiges/link.php?link=yt",
```
Comment (JakobBruening) [inline minecraft_servers/fastmc/manifest.json:16]: Please use lowercase.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress your images.

File: minecraft_servers/fastmc/manifest.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  "direct_ip": "play.fastmc.net",
+  "server_wildcards": [
+    "%.fastmc.net"
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://www.fastmc.net",
+    "web_shop": "https://shop.fastmc.net",
```
Comment (JakobBruening) [inline minecraft_servers/fastmc/manifest.json:13]: The shop is the same page as the normal page, isn't it?

File: minecraft_servers/fastmc/manifest.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    "twitter": "FastMC_NET",
+    "tiktok": "@FastMCNetwork",
+    "youtube": "https://fastmc.net/sonstiges/link.php?link=yt",
+    "discord": "https://discord.fastmc.net",
```
Comment (JakobBruening) [inline minecraft_servers/fastmc/manifest.json:17]: Please use a discord link.
Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/fastmc/manifest.json (hunk 5 above)
Comment (thechriscraft) [reply minecraft_servers/fastmc/manifest.json:13]: Yes, for now, but there will be an update of the store and the website where this is separated with subdomains.

Comment (JakobBruening) [reply minecraft_servers/fastmc/manifest.json:13]: Then please remove the url until you have released the shop.

**Review by JakobBruening — APPROVED** [review]
Thank you!

**Pull request merged**